ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -185,14 +207,35 @@ sub process_path {
 			open TMP, ">>$path";
 			close TMP;
 			if (! -f $path) {
-				$self->error("error creating file $path");
+				$self->error("Error creating file $path");
 				return 0;
 			}
 		} elsif ($type eq "d") {
+            # Should we check if the needed mount is actually mounted?
```

Comment:
It was quite a mix of tabs and spaces, so I've replaced all the tabs with 4 spaces.